e2e/cypress/integration/integrations/slash_commands_spec.js

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @integrations
+
+describe('Integrations', () => {
+    let user1;
+    let user2;
+    let team1;
+    let team2;
+    let testChannelUrl1;
+    let testChannelUrl2;
+    const commandTrigger = 'test-ephemeral';
+    const commandURL = 'http://hidden-peak-21733.herokuapp.com/test_ephemeral';
```

Comment:
I'd suggest not to use this remote URL. Can we use `/send_message_to_channel` endpoint from [local server](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/webhook_serve.js#L27)?

Reply:
I used this because it was stated in the test steps.
I am not completely sure that function behaves as it is expected from the command, but I can make sure, and if not, create a new one along those.